### PR TITLE
multiple access points added

### PR DESCRIPTION
**Pull request by ka0n**
added the possibility to use the scripts on multiple access points by adding a settings variable ap_name. The parameter is prefixed to the mac address. If left empty or variable omitted, mac address not prefixed.

There might be a more elegant way to add this feature - I am a python noob. Thank you for your useful script!

### CODE REVIEW

File: presence-detector.py [hunk 1]
```diff
@@ -96,6 +96,10 @@ def polling_loop(self):
                 if process.returncode == 0:
                     clients = json.loads(process.stdout)
                     for client in clients['clients']:
+                        # Add ap prefix if ap_name defined in settings
+                        if bool(self.settings.ap_name):
```
Comment (rmoesbergen) [inline presence-detector.py:100]: Can you adjust this as follows? I like to use f"" for readability. Also, you test the ap_name variable directly as both None or "" evaluate to False.
```suggestion
                        if self.settings.ap_name:
                            client = f"{self.settings.ap_name}_{client}"
```

**Comment by rmoesbergen** [comment]
Hi @ka0n, thanks for the PR! Sounds like a useful feature. I added one comment to the change, if you resolve that I'll merge.

**Comment by ka0n** [comment]
thanks for your comments @rmoesbergen

File: presence-detector.py [hunk 2]
```diff
@@ -96,6 +96,11 @@ def polling_loop(self):
                 if process.returncode == 0:
                     clients = json.loads(process.stdout)
                     for client in clients['clients']:
+                        # Add ap prefix if ap_name defined in settings
+                        if self.settings.ap_name:
+                            client = f"{self.settings.ap_name}_{client}"
+                            ap = self.settings.ap_name + '_'
```
Comment (rmoesbergen) [inline presence-detector.py:102]: These 2 lines can now be deleted
```suggestion
```

**Comment by ka0n** [comment]
Just realized these two lines were still in

File: presence-detector.py [hunk 3]
```diff
@@ -96,6 +96,11 @@ def polling_loop(self):
...
+                            client = ap + client
```
Comment (rmoesbergen) [inline presence-detector.py]: These 2 lines can now be deleted
```suggestion
```

File: presence-detector.py [hunk 4]
```diff
@@ -96,6 +96,10 @@ def polling_loop(self):
                 if process.returncode == 0:
                     clients = json.loads(process.stdout)
                     for client in clients['clients']:
+                        # Add ap prefix if ap_name defined in settings
+                        if self.settings.ap_name:
+                            client = f"{self.settings.ap_name}_{client}"
+                            client = ap + client
```
Comment (rmoesbergen) [inline presence-detector.py:102]:
```suggestion
```
